internal/querynode/reduce_test.go

```diff
@@ -28,36 +29,122 @@ import (
 	"github.com/milvus-io/milvus/internal/proto/milvuspb"
 )
 
-func TestReduce_AllFunc(t *testing.T) {
-	collectionID := UniqueID(0)
-	segmentID := UniqueID(0)
-	collectionMeta := genTestCollectionMeta(collectionID, false)
+//func TestReduce_AllFunc(t *testing.T) {
```

Comment:
Can this test be rewritten? It is better not to lower the test range.

Reply:
no, `FillTargetEntry` would not assign the row_data member, so old test would crash

Reply:
> 

I means keep this test ,but rewrite with your new code .

Reply:
got it